alluxiofs/client/worker_ring.py

```diff
@@ -205,37 +193,27 @@ def _get_etcd_client(self):
 class ConsistentHashProvider:
     def __init__(
         self,
-        etcd_hosts=None,
-        etcd_port=None,
-        worker_hosts=None,
-        worker_http_port=None,
-        options=None,
-        logger=None,
-        etcd_refresh_workers_interval=None,
-        hash_node_per_worker=None,
-        max_attempts=100,
+        config: AlluxioClientConfig,
+        logger: Optional[logging.Logger] = None,
     ):
         self._logger = logger or logging.getLogger("ConsistentHashProvider")
-        self._etcd_hosts = etcd_hosts
-        self._etcd_port = etcd_port
-        self._options = options
-        self._hash_node_per_worker = hash_node_per_worker
-        self._max_attempts = max_attempts
+        self._config = config
```

Comment:
I feel like we might still want to unwrap the config here like `self._etcd_hosts = etcd_hosts ` and use the member value directly, like what we did in EtcdClient. What do you think